lib/vcr_shared_types/include/shared_types.h

```diff
@@ -0,0 +1,78 @@
+#ifndef __SHARED_TYPES_H__
```

Comment:
Why do we have a VCR shared types in addition to shared_types? Will these eventually be moved upstream?